test/validation/org/apache/cassandra/bridges/ccmbridge/CCMBridge.java

```diff
@@ -48,13 +56,19 @@ public CCMBridge(Config config)
 
     public CCMBridge(int nodeCount)
     {
+        testNodeCount = nodeCount;
         this.ccmDir = Files.createTempDir();
         execute("ccm create %s -n %d --install-dir %s", DEFAULT_CLUSTER_NAME, nodeCount, CASSANDRA_DIR);
     }
 
-    public void destroy()
+    public void captureLogs(String testName)
     {
         stop();
+        preserveClusterLogs(testName);
+    }
+
+    public void destroy()
+    {
```

Comment:
While that is true, you're still breaking the functionality of destroy. Someone should feel comfortable calling destroy on the cluster without needing to call captureLogs or stop beforehand. It would be best to leave `stop` inside `destroy`. If you want to stop before capturing logs at the end of a test, just call the `stop` method, rather than embedding it inside captureLogs.